ui/app/src/components/EditGroupDialog/EditGroupDialog.tsx

```diff
@@ -23,7 +23,7 @@ export function EditGroupDialog(props: EditGroupDialogProps) {
   const { group, onGroupDeleted, onGroupSaved, onSubmittingAndOrPendingChange, ...rest } = props;
 
   return (
-    <EnhancedDialog omitHeader {...rest}>
+    <EnhancedDialog omitHeader maxWidth="lg" {...rest}>
```

Comment:
Does everything continues to work without this change?
```suggestion
    <EnhancedDialog omitHeader {...rest}>
```

Reply:
not needed anymore